src/components/_General/_FormikElements/MultiKeyValue.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { FieldHookConfig, FieldArray, useField, Field } from 'formik';
+import { Button } from 'components/_General/buttons';
+import { Columns, Column } from 'components/_General/Grid';
+import styled from '@emotion/styled';
+
+import Icon from 'components/_General/_UIElements/Icon';
+import PlusIcon from 'assets/Plus.svg';
+import MinusIcon from 'assets/Minus.svg';
+import { inputStyles } from './common';
+
+import FieldContainer from './FieldContainer';
+
+const Input = styled(Field)`
+  ${inputStyles}
+`;
+
+const CustomPaddingColumn = styled(Column)`
+  padding-right: 0;
+  padding-bottom: 0.15em;
+`;
+
+interface MultiKeyValueProps {
+  label?: string;
+  help?: string;
+}
+
+const MultiKeyValue: React.FC<MultiKeyValueProps & FieldHookConfig<string>> = ({
+  label,
+  help,
+  ...props
+}) => {
+  const [field] = useField(props);
+
+  return (
+    <FieldContainer label={label} help={help} {...props}>
+      <FieldArray
+        name={props.name}
+        render={({ push, remove }) => (
+          <div>
+            {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
+            {(field.value as unknown as Array<any>)?.map((_v, index) => (
```

Comment:
Rather than using `any` and suppressing the error/warning, you can just use `unknown`.